src/Akka.Streams.Kafka/Stages/Consumers/Abstract/SubSourceLogic.cs

```diff
@@ -147,23 +201,68 @@ protected void ConfigureSubscription(Action<IImmutableSet<TopicPartition>> parti
 
             IPartitionEventHandler CreateRebalanceListener(IAutoSubscription subscription)
             {
-                return new PartitionEventHandlers.Chain(subscription.PartitionEventsHandler.GetOrElse(PartitionEventHandlers.Empty.Instance), new PartitionEventHandlers.AsyncCallbacks(subscription, SourceActor.Ref, partitionsAssignedCb,
-                    partitionsRevokedCb));
+                return new PartitionEventHandlers.Chain(
+                    subscription.PartitionEventsHandler.GetOrElse(PartitionEventHandlers.Empty.Instance),
+                    new PartitionEventHandlers.AsyncCallbacks(subscription, SourceActor.Ref, partitionsAssignedCb,
```

Comment:
Works the same way that the `BaseSingleSourceStage` / `SingleSourceStage` both do - chains the `AsyncCallback` `IPartitionEventHandler` along with any user-defined ones.